internal/app/shortenerprovider/delete_urls.go

```diff
@@ -0,0 +1,34 @@
+package shortenerprovider
+
+import (
+	"context"
+	"fmt"
+
+	"shorturl/pkg/pgclient"
+)
+
+func (p *ShortenerDBProvider) DeleteURLs(
+	ctx context.Context,
+	tx pgclient.Transaction,
+	urls *[]string,
+	userID string,
+) error {
+	rows, err := p.conn.NamedQueryxContext(
+		ctx,
+		"DeleteURLs",
+		p.mapper.URLIDs(urls),
+		tx,
+		userID,
+	)
+	if err != nil {
+		return fmt.Errorf("can't execute DeleteURLs: %w", err)
+	}
+
+	err = rows.Err()
+
+	if err != nil {
```

Comment:
```suggestion
	if rows.Err() != nil {
```
опять, уверен что нужна эта переменная?

Reply:
Да, потому что тебе два раза надо делать rows.Err() тогда. Выглядит очень некрасиво

```
	err = rows.Err()
	if err != nil {
		return models.URLID{}, fmt.Errorf("can't execute GetURL: %w", err)
	}
```

```
	if rows.Err() != nil {
		return models.URLID{}, fmt.Errorf("can't execute GetURL: %w", rows.Err())
	}
```